ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -838,4 +838,45 @@ describe('RuntimePanel', () => {
     const wrapper = await component();
     expect(wrapper.find('[data-test-id="runtime-status-icon"]').first().prop('src')).toBe(`${iconsDir}/compute-none.svg`);
   });
+
+  it('should prevent runtime creation when disk size is invalid', async() => {
+    runtimeApiStub.runtime = null;
+    act(() => { runtimeStore.set({runtime: null, workspaceNamespace: workspaceStubs[0].namespace}) });
+    const wrapper = await component();
+    await mustClickButton(wrapper, 'Customize');
+    const getCreateButton = () => wrapper.find({'aria-label': 'Create'}).first();
+
+    await pickMainDiskSize(wrapper, 49);
+    expect(getCreateButton().prop('disabled')).toBeTruthy();
+
+    await pickMainDiskSize(wrapper, 4900);
+    expect(getCreateButton().prop('disabled')).toBeTruthy();
+
+    await pickMainDiskSize(wrapper, 50);
+    await pickComputeType(wrapper, ComputeType.Dataproc);
+    await pickWorkerDiskSize(wrapper, 49);
+    expect(getCreateButton().prop('disabled')).toBeTruthy();
+
+    await pickWorkerDiskSize(wrapper, 4900);
+    expect(getCreateButton().prop('disabled')).toBeTruthy();
+  });
+
+  it('should prevent runtime update when disk size is invalid', async() => {
```

Comment:
I'd also add an additional step to both of the steps of setting a valid disk size and observing that the button is enabled.

Reply:
good idea, will do